projects.md

```diff
@@ -1,51 +1,13 @@
-
-Each project will consist of a complete minimum viable product (MVP). The first round of MVPs can be for the students' own projects or will be suggested by us. [Subsequent projects](http://foundersandcoders.org/mvp) will be for external clients.
-
-##Guidelines
-
-### Project selection
-
-+ After a project is posted on the MVP projects board, students can add their names to it.
-+ Students will work on projects in teams of 2-5.
-
-### Process
-
-+ Each project will normally start on a Monday and end on a Friday two weeks later.
-+ Each team will use PivotalTracker to manage the project.
-+ Each team will appoint a trainer to train the client in using PivotalTracker.
-+ Each project will consist of two one-week sprints.
-+ The client will be invited to join the team on the first Monday to agree the scope of the project and to participate in a sprint planning meeting.
-+ The client will be invited to join the team on the second Monday for a sprint review and planning meeting.
-+ The client will be invited to join the team for a project review on the final day of the project.
-
-### Communication
-
-+ Each project team will appoint a client manager, who's job it will be to communicate with the client throughout the project.
-+ It is up to the client manager to arrange meeting times with the client.
-+ It is up to the client manager to notify the client of any changes to schedule or personnel.
-+ It is up to the client manager to delegate any of these tasks if they are not available to carry them out.
-+ It is the responsiblity of all team members to notify their colleagues of any changes to their own availability.
-
-### Documentation
-
-+ Each team will maintain a GitHub repository for the project with a README.
-+ Unless the client requests otherwise, the repo will be public.
-+ But the repo and the README should not refer to the client by name.
-+ No sensitive client information should be included in the repo.
-+ Each team will complete a screencast for the client to accompany the README, describing the project and how to install and use it.
-
-### Pay
-
-All income received for these MVPs will be divided equally between the students.
-
-### On completion of four projects
-
-After an MVP project is completed, students may continue with any follow-up work for the client at their own discretion.
-
-They may also pick up further MVP projects, if any are available.
+**First, agree the scope**     
+Before you do anything, agree the scope of the project.  
 
+**Second, create the README**     
+Next, create a repo with a README to describe the project.  
 
+**Third, create the wireframes and user stories**     
+Use Google Slides for wireframes and PivotalTracker for user stories.
 
+**Fourth, create acceptance tests**     
+Don't start work on the project until you have created a set of failing tests that reflect the user stories.
```

Comment:
@sofer Reckon we should clarify here that this should be one test -> watch it fail -> write the code to pass. Rinse & repeat (rather than all the tests for the user stories at once) as we discussed a little while back :)